src/lib/setup_sharify.js

```diff
@@ -46,6 +46,8 @@ sharify.data = _.extend(
     "ENABLE_EXPERIMENTAL_STITCH_INJECTION",
     "EOY_2016_ARTICLE",
     "EOY_2016",
+    "EOY_2018_1",
+    "EOY_2018_2",
```

Comment:
We chatted about make the naming for these keys a little more explicit but I think that can definitely happen in a refactor outside of this PR.